kafka-ui-contract/src/main/resources/swagger/kafka-ui-api.yaml

```diff
@@ -750,15 +750,28 @@ paths:
           required: true
           schema:
             type: string
+        - name: page
+          in: query
+          required: false
+          schema:
+            type: integer
+        - name: perPage
+          in: query
+          required: false
+          schema:
+            type: integer
+        - name: search
+          in: query
+          required: false
+          schema:
+            type: string
       responses:
         200:
           description: OK
           content:
             application/json:
               schema:
-                type: array
-                items:
-                  $ref: '#/components/schemas/SchemaSubject'
+                $ref: '#/components/schemas/SchemaSubjectsResponse'
```

Comment:
since this is breaking change, we can
1. do UI work within this PR also and move to new API simultaneously
2. create new endpoint, merge this PR and create new for UI

@Haarolean what do you think? I would prefer 1. because UI implementation can have a big lag, and it is not very comfortable to have many opened PRs

Reply:
We'll do both backend and frontend within a same PR.